Apps/Src/Tasks.c

```diff
@@ -103,17 +105,17 @@ void throwTaskError(error_code_t errorCode, callback_t errorCallback, error_sche
 
             #if DEBUG == 1
                // Print the error that caused this fault
-                printf("\n\rCurrent Error Code: 0x%04x\n\r", errorCode);
+                // printf("\n\rCurrent Error Code: 0x%04x\n\r", errorCode);
```

Comment:
why not just delete this

Reply:
I suppose we might want to maybe enable it later when debugging in the future. I would consider this to be apart of the code cleanup PR

Reply:
Can we make an issue ticket to make this conditional on a separate macro like PRINT_ERROR or something?

Reply:
Yes